README.md

```diff
@@ -1,2 +1,298 @@
+   - Update storage cvg devices for data and metadata with respect to the devices in your environment.
+   - Update nodes section with proper node hostnames from your Kubernetes cluster.
+     - If the Kubernetes control plane nodes are required to be used for deployment, make sure to remove the taint from it before deploying CORTX.
+     - For further details and reference, you can view the official Kubernetes documentation topic on [Taints & Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+   - For further details on `solution.yaml` specifics, review the [Solution YAML Overview](#solution-yaml-overview) below.
+
+
+3. Run the `deploy-cortx-cloud` script, passing in the path to your updated `solution.yaml` file.
+
+   ```bash
+   ./deploy-cortx-cloud.sh solution.yaml
+   ```
+
+4. Validate CORTX on Kubernetes statues
+
+   ```bash
+   DATA_POD=$(kubectl get pods -l cortx.io/service-type=cortx-data --no-headers | awk '{print $1}' | head -n 1)
+   kubectl exec -it $DATA_POD -c cortx-hax -- /bin/bash -c "hctl status"
+   ```
+
+### Upgrading CORTX on Kubernetes
+
+> **NOTE:** As the CORTX on Kubernetes architecture is evolving, the upgrade path for CORTX on Kubernetes is evolving as well. As a workaround until more foundational upgrade capabilities exist, the following steps are available to manually upgrade your CORTX on Kubernetes environment to a more recent release.
+
+1. Deploy CORTX on Kubernetes according to the [Deploying CORTX on Kubernetes](#deploying-cortx-on-kubernetes) steps above.
+
+2. Shutdown all CORTX Pods
+
+   ```bash
+   ./shutdown-cortx-cloud.sh solution.yaml
+   ```
+
+3. Patch the CORTX on Kubernetes Deployments using an updated image _(NOTE: You will want to update the `TARGET_IMAGE` variable below to your desired image tag)_
+
+   ```bash
+   TARGET_IMAGE="ghcr.io/seagate/cortx-all:2.0.0-593-custom-ci"
+   for DEPLOYMENT in $(kc get deploy | grep cortx | awk '{print $1}')
+   do
+      kc set image deployment/$DEPLOYMENT "*=$TARGET_IMAGE"
+   done
+   ```
+
+4. Start all CORTX Pods
+
+   ```bash
+   ./start-cortx-cloud.sh solution.yaml
+   ```
+
+### Using CORTX on Kubernetes
+
+**_TODO_** Port https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/754155622/CORTX+Kubernetes+N-Pod+Deployment+and+Upgrade+Document+using+Services+Framework#5.-Understanding-Management-and-S3-Endpoints-and-configuring-External-Load-balancer-service(Optional) here or into a linked `doc/readme` file.
+
+### Log collection for CORTX on Kubernetes
+
+To gather logs from a CORTX on Kubernetes deployment, run the `logs-cortx-cloud` script while passing in the `solution.yaml` file to it.
+
+```bash
+./logs-cortx-cloud.sh --solution-config solution.yaml
+```
+
+### Undeploying CORTX on Kubernetes
+
+Run the `destroy-cortx-cloud` script, passing in the path to the previously updated `solution.yaml` file
+
+```bash
+./destroy-cortx-cloud.sh solution.yaml
```

Comment:
Unrelated comment: some scripts use a position argument for the yaml file, and others require `-s/----solution-config`, so it's a bit inconsistent.

Reply:
Agreed. It's a bit of an issue when all the scripts are created ad-hoc without a lot of long-term planning. We will look to consolidate some of this and standardize going forward.